app/src/main/java/com/google/samples/apps/topeka/widget/AvatarView.kt

```diff
@@ -95,7 +95,7 @@ class AvatarView @JvmOverloads constructor(
         super.onDraw(canvas)
         if (isChecked) {
             ContextCompat.getDrawable(context, R.drawable.selector_avatar).apply {
-                setBounds(0, 0, width, height)
+                this!!.setBounds(0, 0, width, height)
```

Comment:
is this necessary? :thinking: if `setBounds` is an instance method, then the instance can't be null.

Wondering whether the addition of `this` and `!!` throughout the PR is needed.